### PR TITLE
Javascript/javascript1/week3

### PR DESCRIPTION
**Pull request by Mamidi001**

### CODE REVIEW

File: javascript/javascript1/week2/index.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const fullname2= getFullname('swetha','mamidi');
+console.log (`FamilyName : ${fullname1}`);
+console.log(`FirstName:${fullname2}`);
+*/
+//function that add onemore parameter
+
+function getFullname(firstname, surname, useFormalName, gender){
+    let fullName= firstname+" " +surname;
+    
+    if(firstname===" "&& surname===" "){
```
Comment (aina21) [inline javascript/javascript1/week2/index.js:18]: Do you mean `firstname===""` ?

**Review by aina21 — CHANGES_REQUESTED** [review]
Remove irrelevant files of week2 from this PR.

File: javascript/javascript1/week3/index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      minutes: 0,
+    },
+    {
+      title: "The Wire",
+      days: 2,
+      hours: 12,
+      minutes: 0,
+    },
+  ];
+  //replace with different index in the array
```
Comment (aina21) [inline javascript/javascript1/week3/index.js:23]: it's better to have a function to get an id and values to replace.

File: javascript/javascript1/week3/index2.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+
+//Smart phone usage app
+
+    const activities = [];
+    
+    function addActivity(date, activity,duration){
+        const object ={
```
Comment (aina21) [inline javascript/javascript1/week3/index2.js:7]: object is not a good name

File: javascript/javascript1/week3/index3.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  
+  const namesToRemove = names.indexOf("Ahmad");
+  //to remove specific element 
+  names.splice(1,1);
```
Comment (aina21) [inline javascript/javascript1/week3/index3.js:17]: names.splice(namesToRemove,1)

File: javascript/javascript1/week3/index4.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+ // travel time calculation
+ const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  const travelTime = getTravelTime(travelInformation);
+  function getTravelTime(){
+      let journeyDuration = travelInformation.destinationDistance/ travelInformation.speed;//8.64
```
Comment (aina21) [inline javascript/javascript1/week3/index4.js:8]: const

**Review by aina21 — CHANGES_REQUESTED** [review]
good job :+1:

File: javascript/javascript1/week3/index3.js (hunk 4 above)
Comment (Mamidi001) [reply javascript/javascript1/week3/index3.js:17]: Ok i will change.

File: javascript/javascript1/week3/index4.js (hunk 5 above)
Comment (Mamidi001) [reply javascript/javascript1/week3/index4.js:8]: Done

File: javascript/javascript1/week3/index2.js (hunk 3 above)
Comment (Mamidi001) [reply javascript/javascript1/week3/index2.js:7]: changed the object as activityInfo

**Pull request merged**